### ISSUE TITLE
Question: Are activation scripts compatible with `noarch`?

### ISSUE THREAD

**Issue by mfansler** [conda/conda-build#4868]
Can a package that does not require compilation, but has platform-specific activation scripts be `noarch`? That is, if we have both a `.sh` and `.bat` activation script, can we simply copy both to `$PREFIX/etc/conda/activate.d/`, will `conda activate` automatically only use the one specific to the current platform?

Or, must one only copy the `.sh` on Unix systems and `.bat` on Windows system, and therefore need platform-specific package builds?

**Comment by jaimergp** [comment]
Yes, it should work. Just make sure to include all the scripts (for all platforms, that is) in the (de)activate.d directories, and you should be good. conda will pick the right ones on each platform.

**Comment by travishathaway** [comment]
Looks like the question has been answered. Closing for now.
